### PR TITLE
added seeds for retrospective assessment

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: seeds/data/projects.json [hunk 1]
```diff
@@ -300,5 +300,86 @@
     "licenceNumber": "PR-X00005",
     "licenceHolderId": "5b7bad13-f34b-4959-bd08-c6067ae2fcdd",
     "status": "inactive"
+  },
+  {
+    "id": "b8973df0-408c-4f3c-a7f2-b5b8f8c2943b",
+    "title": "RA true",
+    "schemaVersion": 1,
+    "status": "active",
```
Comment (lennym) [inline seeds/data/projects.json]: I think setting active projects without a grant date or expiry date might cause some weirdness down the road.

File: seeds/data/projects.json [hunk 2]
```diff
@@ -300,5 +300,86 @@
...
+    "licenceHolderId": "5b7bad13-f34b-4959-bd08-c6067ae2fcdd",
+    "establishmentId": 8201,
+    "licenceNumber": "PR-X10007"
+  },
+  {
+    "id": "b0901bea-7e40-4d43-bcd9-07ad136ddbcd",
+    "title": "RA legacy true",
+    "schemaVersion": 1,
```
Comment (wheelsandcogs) [inline seeds/data/projects.json]: Legacy? Should this be `0`?

File: seeds/data/projects.json [hunk 3]
```diff
@@ -300,5 +300,86 @@
...
+    "status": "active",
+    "licenceHolderId": "5b7bad13-f34b-4959-bd08-c6067ae2fcdd",
+    "establishmentId": 8201,
+    "licenceNumber": "PR-X10008"
+  },
+  {
+    "id": "a395a905-c2c6-4ea8-bb9f-366c188309c3",
+    "title": "RA legacy false",
+    "schemaVersion": 1,
```
Comment (wheelsandcogs) [inline seeds/data/projects.json]: Same here `"schemaVersion": 0,` ?

File: seeds/data/projects.json (hunk 1 above)
Comment (wheelsandcogs) [reply seeds/data/projects.json]: Looks like this was resolved

**Pull request merged**